### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by rychrr**
Hi, 

I have updated the portfolio  desktop version  and also made adjustment to improve responsiveness of the page

Regards,

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -481,6 +464,336 @@ textarea::placeholder {
   color: #172b4d;
 }
```
Comment (zieeco) [inline style.css:466]: ### contact section desktop version


- Hey, let's align the content of this section on the desktop version to be close to the Figma design to make our design consistent. see the images below to improve your design:

*Your design*
image


*Figma design*
image

File: style.css [hunk 2]
```diff
@@ -481,6 +464,336 @@ textarea::placeholder {
...
+}
+
+button:disabled {
+  color: #5b5957;
+  background: #f4f1f0;
+}
+
 footer {
   width: 100vw;
   display: flex;
```
Comment (zieeco) [inline style.css:799]: - Hey, let's also reduce the spaces in between each icon to improve our design as compared with the Figma design:

*Your design*
image

*figma design*
image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @rychrr,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -315,29 +321,6 @@ body {
...
-}
-
-button:disabled {
-  color: #5b5957;
-  background: #f4f1f0;
-}
-
 /* Work skills , language and framework */
 .work_skills {
   width: 100vw;
```
[the remark this answers is not in the document]
Comment (rychrr) [reply style.css]: Thank you this was resolved

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @rychrr,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**